pkg/scalers/gcp_pubsub_scaler.go

```diff
@@ -60,6 +61,11 @@ func NewPubSubScaler(config *ScalerConfig) (Scaler, error) {
 		return nil, fmt.Errorf("error parsing PubSub metadata: %s", err)
 	}
 
+	meta.aggregation, err = NewStackdriverAggregator(60, "mean", "none")
```

Comment:
Just to clarify for PubSub it doesn't make sense to support this feature, right?